tests/test_basic_correctness.py

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright (c) 2025 Huawei Technologies Co., Ltd. All Rights Reserved.
+# This file is a part of the vllm-ascend project.
+# Adapted from vllm/tests/basic_correctness/test_basic_correctness.py
+# Copyright 2023 The vLLM team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Compare the short outputs of HF and vLLM when using greedy sampling.
+
+Run `pytest tests/basic_correctness/test_basic_correctness.py`.
+"""
+import os
+
+import pytest
+from conftest import VllmRunner
+from model_utils import check_outputs_equal
+
+MODELS = ["Qwen/Qwen2.5-7B-Instruct"]
+
```

Comment:
Let's use modelscope for better model download and keep the configuration of `PYTORCH_NPU_ALLOC_CONF` to ensure oom not raised in CI
```suggestion

os.environ["VLLM_USE_MODELSCOPE"] = "True"
os.environ["PYTORCH_NPU_ALLOC_CONF"] = "max_split_size_mb:256"
```

Reply:
I don't know if hfrunner have the same way to speed up model download, otherwise we will download two weights from two different repositories

Reply:
Make sense. We currently disable modelscope then, but if requesting hf to download the model will break CI, we need to fallback to this. hfrunner can use splicing local paths to prevent duplicate downloads.